Makefile

```diff
@@ -36,6 +36,13 @@ fonts:
 	mkdir -p build/$(browser)/$(type)/public
 	cp -r shared/font build/$(browser)/$(type)/public/
 
+web-resources:
```

Comment:
To avoid making changes to the blocking code that is shared with other clients (Apps), I'm combining all the single surrogate files back into a single surrogates.txt file with the format that the Trackers module expects https://github.com/duckduckgo/privacy-grade/blob/master/src/trackers.js#L32